hiero-enterprise-helidon-me-sample/.env.example

```
@@ -0,0 +1,3 @@
+hiero.accountId=${HEDERA_ACCOUNT_ID}
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The hiero.accountId key has incorrect delimiter

(IncorrectDelimiter)